lib/shelley/test/data/plutus/ping-pong_1-1.bin

```diff
@@ -1 +1 @@
-84a30080018183581d714d72cf569a339a18a7d9302313983f56e0d96cd45bdcb1d6512dca6a015820923918e403bf43c34b4ef6b48eb2ee04babed17320d8d1b9ff9ad086e86f44ec0200a10481d87980f5f6
\ No newline at end of file
+84a300800181a300581d714d72cf569a339a18a7d9302313983f56e0d96cd45bdcb1d6512dca6a01010282005820923918e403bf43c34b4ef6b48eb2ee04babed17320d8d1b9ff9ad086e86f44ec0200a10481d87980f5f6
```

Comment:
Needed to make `updateSealedTxSpec` pass: now that we're ending up serialising to Babbage txs the normal form is slightly different.